tls/s2n_signature_scheme.c

```diff
@@ -426,24 +376,21 @@ const struct s2n_signature_preferences s2n_signature_preferences_20210816 = {
 };
 
 const struct s2n_signature_scheme* const s2n_sig_scheme_pref_list_rfc9151[] = {
-    /* ECDSA - TLS 1.3 */
-    &s2n_ecdsa_secp384r1_sha384,
+    /* ECDSA */
+    &s2n_ecdsa_sha384,
 
     /* RSA PSS - TLS 1.3 */
     &s2n_rsa_pss_pss_sha384,
 
-    /* ECDSA - TLS 1.2 */
-    &s2n_ecdsa_sha384, /* same iana value as TLS 1.3 s2n_ecdsa_secp384r1_sha384 */
-
     /* RSA */
     &s2n_rsa_pss_rsae_sha384,
 
     &s2n_rsa_pkcs1_sha384,
 };
 
 const struct s2n_signature_scheme* const s2n_cert_sig_scheme_pref_list_rfc9151[] = {
-    /* ECDSA - TLS 1.3 */
-    &s2n_ecdsa_secp384r1_sha384,
+    /* ECDSA */
+    &s2n_ecdsa_sha384,
```

Comment:
If you expand the diff, there were no other options between s2n_ecdsa_secp384r1_sha384 and s2n_ecdsa_sha384. Just a big comment.